tests/installation/partitioning.pm

```diff
@@ -20,6 +20,15 @@ use partition_setup qw(%partition_roles is_storage_ng_newui);
 
 sub run {
     assert_screen 'partitioning-edit-proposal-button', 40;
+    if (get_var("PARTITION_EDIT")) {
```

Comment:
I think var PARTITIOIN_EDIT don't have the meaning of "set root as ext4 and set home as btrfs", how about check var like check_var('PARTITION_ROOT', 'ext4') and check_var('PARTITION_HOME', 'btrfs'), or check_var('PARTITIOIN_ROOT_HOME', 'ext4_btrfs');

And it's better to involve this script maintainer Joachim into reviewer.